Source/PlasticSourceControl/Private/PlasticSourceControlOperations.cpp

```diff
@@ -164,17 +164,17 @@ bool FPlasticConnectWorker::Execute(FPlasticSourceControlCommand& InCommand)
 		else
 		{
 #if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION >= 2
-			const FText ErrorText(LOCTEXT("NotAPlasticRepository", "Failed to enable Plastic SCM revision control. You need to create a Plastic SCM workspace for the project first."));
+			const FText ErrorText(LOCTEXT("NotAPlasticRepository", "Failed to enable Unity Version Control revision control. You need to create a Unity Version Control workspace for the project first."));
 #else
-			const FText ErrorText(LOCTEXT("NotAPlasticRepository", "Failed to enable Plastic SCM source control. You need to create a Plastic SCM workspace for the project first."));
+			const FText ErrorText(LOCTEXT("NotAPlasticRepository", "Failed to enable Unity Version Control source control. You need to create a Unity Version Control workspace for the project first."));
```

Comment:
"Unity Version Control revision control" sounds a bit reiterative, doesn't it? As this is a message displayed to the user, we may want to give it a thought.

Some alternatives could be:
- Failed to enable Unity Version Control. You need to create a Unity Version Control workspace for the project first.
- Failed to enable Unity Version Control plugin. You need to create a Unity Version Control workspace for the project first.
- Failed to enable revision control. You need to create a Unity Version Control workspace for the project first.